### PR TITLE
[docs] Magma Header Update to match Magmacore.org

### PR DESCRIPTION
**Pull request by jimmytipit**
## Summary

This PR addresses the need to unify magma docs layout with magmacore.org site .

## Test Plan

A demo of the site changes can be reviewed at https://competent-euler-513227.netlify.app/docs/basics/introduction.html 

## Additional Information
- Creates a unified design 
- Swaps docusaurus logo for magma
- Still "to do" - adjusting footer links/colors

### CODE REVIEW

File: docs/docusaurus/siteConfig.js
Comment (xjtian) [inline docs/docusaurus/siteConfig.js:50]: delete

File: docs/docusaurus/siteConfig.js
Comment (xjtian) [inline docs/docusaurus/siteConfig.js:46]: do we not need this anymore?

File: docs/docusaurus/siteConfig.js
Comment (xjtian) [inline docs/docusaurus/siteConfig.js:38]: I'm worried about breaking existing links with these changes. Is the plan to keep publishing on gh-pages for now? Or should we coordinate a change to gh-pages to redirect

**Review by xjtian — CHANGES_REQUESTED** [review]
Looks fancy! Just a question on my end r.e. link rot

File: docs/docusaurus/siteConfig.js
Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:38]: We should probably coordinate a time to redirect.  Definitely do not want to break anything until we are prepared to move it over.  We're working on a CNAME for the docs to live at docs.magmacore.org.  I'm working with @jbryce on this one, so I'd say let's hold until then?

Comment (xjtian) [reply docs/docusaurus/siteConfig.js:38]: Cool, just give me a ping when we're good to go on that front and I'll push the redirect to gh-pages.

File: docs/docusaurus/siteConfig.js
Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:46]: We won't need that for the Netlify instance b/c we're pointing directly to the docs directory.

File: docs/docusaurus/siteConfig.js
Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:38]: @xjtian Thanks again for reviewing.  I've fixed our DNS issues and I believe we're ready to land this.  Once we land it, we will be using docs.magmacore.org.

Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:38]: One thing... once we make the switch, Magma will want to update github pages to redirect to docs.magmacore.org (https://docs.github.com/en/enterprise/2.13/user/articles/redirects-on-github-pages)

Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:38]: Hi @xjtian - Just a ping on the above ^^ :)

Comment (xjtian) [reply docs/docusaurus/siteConfig.js:38]: do we have CI set up to automatically deploy to docs.magmacore.org? I see 1.1.0 as the most recent available version on that site but our docusaurus is up to 1.3

**Comment by xjtian** [comment]
I've disabled our gh-pages docusaurus job on circleci for now, but we shouldn't land this until there's a CD pipeline into docs.magmacore.org for doc changes.

File: docs/docusaurus/siteConfig.js
Comment (jimmytipit) [reply docs/docusaurus/siteConfig.js:38]: The demo version (https://competent-euler-513227.netlify.app/docs/basics/introduction.html) is pointing to the docs site that I forked on my own repo.  Thus the discrepancy.  I don't have access to approve netlify pointing to the magma repo. so we'll need @jbryce to work with me on that.

**Comment by jimmytipit** [comment]
@xjtian @jbryce I just set up our Netlify CD to point to the live Docs repo.  However, b/c of the siteConfig.js changes in this PR, the build isn't going to go through without this pull request being approved.  Not ideal, I realize.

**Comment by jimmytipit** [comment]
Closing this and redoing the PR.

**Pull request closed without merging**
